getpaid/backends/pagseguro/tests.py

```diff
@@ -0,0 +1,16 @@
+"""
+This file demonstrates writing tests using the unittest module. These will pass
+when you run "manage.py test".
+
+Replace this with more appropriate tests for your application.
+"""
+
+from django.test import TestCase
+
+
+class SimpleTest(TestCase):
+    def test_basic_addition(self):
+        """
+        Tests that 1 + 1 always equals 2.
+        """
+        self.assertEqual(1 + 1, 2)
```

Comment:
@nielsonsantana all the existing backends have tests inside `getpaid_test_project` so I think it would make sense to add it there too.

I think your tests should work if you did it they way it is in`.travis.yml` but if you have more issies with it please add a question/issue because it may not be related to this PR.